src/pages/ContentScripts/index.scss

```diff
@@ -45,8 +45,9 @@
 
 .floating-window {
   opacity: 1 !important;
-  padding: 8px 8px !important;
+  padding: 12px 4px 8px 4px !important;
   border-radius: 6px !important;
+  box-shadow: 0 2px 16px rgba(0, 0, 0, 0.15) !important;
```

Comment:
Hi, @wj23027, by [the UI design](https://www.figma.com/file/lHPwpACIjHsGQp1sgW4flf/Work-with-Frank?node-id=471%3A0), we may add `box-shadow` in light theme only.

<img width="1327" alt="image" src="https://user-images.githubusercontent.com/32434520/200739836-b1cff6ef-64dd-424d-9aa7-46678a5b15af.png">

Could you put the `box-shadow` line into `&.light{}`？

<img width="560" alt="image" src="https://user-images.githubusercontent.com/32434520/200739972-90a5d19f-2ee2-4453-9e51-84978e12bbcd.png">